lab_3_ann_retriever/main.py

```diff
@@ -194,6 +278,16 @@ def save(self, file_path: str) -> bool:
         Returns:
             bool: True if saved successfully, False in other case
         """
+        if not isinstance(file_path, str):
+            return False
+        vectorizer_info = {
+            "idf_values": self._idf_values,
+            "vocabulary": self._vocabulary,
+            "token2ind": self._token2ind
+        }
+        with open(file_path, "w", encoding="UTF-8") as file:
```

Comment:
saved files contain `\u...` instead of russian letters

Reply:
However, files on my machine contain russian letter now...